packages/app/components/BetaNFTModal.tsx

```diff
@@ -0,0 +1,81 @@
+import {
+  BodyText,
+  Button,
+  Modal,
+  ModalContent,
+  ModalHeader,
+  HeadingText,
+  ButtonLink,
+  ModalFooter,
+} from "@/ui";
+import { ModalId, useModalContext } from "@/contexts";
+import { useDisconnect } from "wagmi";
+import { DiscordIcon, NFT } from "@/public/assets";
+
+export const BetaNFTModal = () => {
+  const { closeModal, isModalOpen } = useModalContext();
+  const { disconnect } = useDisconnect();
+
+  const isOpen = isModalOpen(ModalId.BETA_NFT);
+
+  const closeAction = () => closeModal(ModalId.BETA_NFT);
+
+  return (
+    <Modal
+      className="bg-green-gradient"
+      maxWidth="xl"
+      isOpen={isOpen}
+      closeAction={closeAction}
+    >
+      <ModalHeader className="flex-col pt-8 pb-4 space-y-2">
+        <HeadingText size={3}>Stackly beta is here</HeadingText>
+        <HeadingText className="text-em-med" weight="medium">
+          Stackly beta is exclusive for our NFT holders.
+        </HeadingText>
+        <Button
+          variant="quaternary"
+          iconLeft="close"
+          size="icon"
+          onClick={closeAction}
+          className="absolute top-5 right-5"
+        />
+      </ModalHeader>
+      <ModalContent className="border-b border-surface-50 py-10 mt-0">
+        <div className="flex flex-col items-center space-y-6">
+          <div className="relative">
+            <div className="absolute -top-3 -right-2 bg-primary-400 rounded-lg px-2 py-1">
+              <BodyText weight="medium" size={1}>
+                FREE
+              </BodyText>
+            </div>
+            <NFT width={135} height={135} className="shadow-2xl" />
+          </div>
+          <HeadingText>Mint your free NFT via our Discord.</HeadingText>
+          <div className="flex flex-col items-center w-full px-6 space-y-2">
+            <ButtonLink
+              href="https://www.discord.com"
+              passHref={true}
+              width="full"
+              target="_blank"
+              iconRight="discord"
+            >
+              Go to Discord
+            </ButtonLink>
+            <BodyText size={2} className="text-em-low">
+              You have another wallet?{" "}
+              <button
+                className="text-black hover:underline"
+                onClick={() => disconnect()}
+              >
+                Disconnect Wallet
+              </button>
+            </BodyText>
+          </div>
+        </div>
+      </ModalContent>
+      <ModalFooter className="flex-col pt-4 pb-8">
+        <BodyText>Made with 💜 from Swapr</BodyText>
```

Comment:
this text is missing some styles, correct size is 1 (default is 2). Proper color.

```suggestion
        <BodyText size={1} className="text-em-med">Made with 💜 from Swapr</BodyText>
```